css/style.css

```diff
@@ -0,0 +1,132 @@
+body {
+  color: #000;
+  overflow-x: hidden;
+  height: 100%;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+}
+.card {
+  padding: 30px 40px;
+  margin-top: 60px;
+  margin-bottom: 60px;
+  border: none !important;
+  box-shadow: 0 6px 12px 0 rgba(0, 0, 0, 0.2);
+}
+.blue-text {
+  color: #00bcd4;
+}
+.form-control-label {
+  margin-bottom: 0;
+}
+input,
+textarea,
+button {
+  padding: 8px 15px;
+  border-radius: 5px !important;
+  margin: 5px 0px;
+  box-sizing: border-box;
+  border: none;
+  border-bottom: 1px solid #ccc;
+  border-radius: 0px !important;
+  font-size: 18px !important;
+  font-weight: 300;
+}
+button {
+  padding: 8px 15px;
+  border-radius: 5px !important;
+  margin: 5px 0px;
+  box-sizing: border-box;
+  border: 1px solid #ccc;
+  font-size: 18px !important;
+  font-weight: 300;
+}
+input:focus,
+textarea:focus {
+  -moz-box-shadow: none !important;
+  -webkit-box-shadow: none !important;
+  box-shadow: none !important;
+  border: 1px solid red;
+  outline-width: 0;
+  font-weight: 400;
+  border-radius: 4px !important;
+}
+.btn-block {
+  text-transform: uppercase;
+  font-size: 15px !important;
+  font-weight: 400;
+  height: 43px;
+  cursor: pointer;
+  padding-top: 10px;
+  border-radius: 5px;
+}
+.btn-block:hover {
+  color: #fff !important;
+}
+button:focus {
+  -moz-box-shadow: none !important;
+  -webkit-box-shadow: none !important;
+  box-shadow: none !important;
+  outline-width: 0;
+}
+#h5ss {
+  background-color: #909090 !important;
+  width: 124px;
+  margin-right: 6px;
+  padding: 7px 2px;
+}
+#mar {
+  background-color: #ab4753;
+  width: 38px;
+  height: 38px;
+}
+#heading {
+  display: flex;
+  position: relative;
+  margin-right: 6px;
+}
+.req {
+  font-size: 17px !important;
+  color: red;
+  margin-left: 16px;
+  display: none;
+}
+#redtext {
+  color: red;
+}
+.colorinput {
+  width: 100%;
+}
+.selectinput {
+  background: transparent;
+  border: none;
+  height: 40px;
+}
+.btn-change6{
+  color: white !important;
+  padding-top: 12px !important;
```

Comment:
why do you need to use important so often try writing CSS that no overlap each other